docs/how-to/integrate-platform.md

```diff
@@ -0,0 +1,65 @@
+# Integrate PMM with Percona Platform
+Percona Platform Portal brings together database distributions, support expertise, services, management, and automated insights. 
+
+Connect your PMM servers to Percona Platform to boost the monitoring capabilities of your PMM installations and manage database deployments easier. In addition, you get access to PMM updates, automated insights, advanced advisor checks and more alert rule templates.
+
+### Connect PMM to Percona Platform
+You can connect to Percona Platform with a Percona Account or via Google or GitHub authentication. If [Percona Support](https://www.percona.com/about-percona/contact) has enabled a custom identity provider for your account, you can also log in using your company's credentials.
+
+We recommend that you connect with a Percona Account, as this gives you access to other Percona services, including Percona Platform, Percona Customer Portal, and Community Forum. If you don’t have a Percona Account, you can create one on the [Percona Platform homepage](https://portal.percona.com/login) using the **Don't have an account? Create one?** link.
+
+#### Pre-requisites
+To ensure that PMM can establish a connection to Percona Platform: 
+
+### Upgrade to PMM 2.27.0 or later
+ Before connecting your PMM server to Percona Platform, make sure you are using PMM version 2.27 or newer. Otherwise, upgrade your PMM installation beforehand. 
```

Comment:
```suggestion
 Before connecting your PMM server to the Percona Platform, make sure you use PMM version 2.27 or newer. Otherwise, upgrade your PMM installation beforehand. 
```